task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -4,6 +4,38 @@ namespace task_DEV_3
 {
     class Program
     {
+        static void InputOfArrayElements(int[] numberArray, int arrayLength)
```

Comment:
You needn't to add numberArray variable. You should change the name of the method and add return value. 
static int[] CreateNumbersArray(int arrayLength). And this method will return the array with numbers.

Reply:
done